### PR TITLE
feat(card): support left and right image placement

### PR DESCRIPTION
**Pull request by toadkicker**
Added img-height and img-width props in case someone wants to break it themselves. Addressed #1958

### CODE REVIEW

**Comment by toadkicker** [comment]
@pi0 or someone who has CI access: can you please trigger a build (or tell me what you prefer to do) so that this will run clean now that NPM has corrected connectivity issues?

File: src/components/card/card-img.js [hunk 1]
```diff
@@ -27,20 +43,27 @@ export const props = {
...
     }
 
     return h(
       'img',
       mergeData(data, {
         staticClass,
-        class: { 'img-fluid': props.fluid },
-        attrs: { src: props.src, alt: props.alt }
+        class: {'img-fluid': props.fluid},
+        attrs: {src: props.src, alt: props.alt, height: props.height, width: props.width}
```
Comment (tmorehouse) [inline src/components/card/card-img.js]: Will this always add a `width` and `height` attributes to the rendered element?

**Comment by tmorehouse** [comment]
You can trigger a rebuild by making a minor edit (such as editing a comment)

File: src/components/card/card.js [hunk 2]
```diff
@@ -52,37 +53,37 @@ export default {
...
-      // Above the header placement.
-      if (props.imgTop || !props.imgBottom) {
-        childNodes.push(img)
+      if (props.imgLeft) {
+        staticClass += ' flex-row'
       }
+      if (props.imgRight) {
+        staticClass += ' flex-row-reverse'
+      }
+      childNodes.push(img)
```
[the remark this answers is not in the document]
Comment (tmorehouse) [reply src/components/card/card.js]: I.e. here is what is rendered when setting the image position to `top` and `bottom`:
image

Note the right hand side card should have the image should be on the bottom.

**Comment by tmorehouse** [comment]
Also appears that the image `overlay` option is no longer working ;-)

image

It appears the `` is not being inserted into the card.

**Pull request merged**